prov/util/src/util_attr.c

```diff
@@ -620,6 +622,33 @@ int ofi_check_ep_attr(const struct util_prov *util_prov, uint32_t api_version,
 		}
 	}
 
+	if (user_attr->max_order_raw_size &&
+	    (user_attr->max_order_raw_size > prov_attr->max_order_raw_size)) {
```

Comment:
nit: here and the 2 if statements that follow can just use the > comparison.  The check for non-zero isn't needed.

Reply:
Thanks